doc/_info_/11logo.order

```diff
@@ -0,0 +1,481 @@
+A GLOSSARY OF 11LOGO BUILT-IN PROCEDURES (PRIMITIVES) 
+--------------------------------------------------------------
+Adapted from A.I. Memo 315 (LOGO Memo 14) of Abelson and Adams.
+
+
+ABSTRACT
+--------
+
+This is a brief description of the built-in procedures (primitives) in 11LOGO 
+version 1005 available on PDP-10/ITS. https://github.com/PDP-10/its.
+This 11LOGO version is sensetive whether you type in UPPER CASE or lower case. 

```

Comment:
I think it would be clearer to say: "This 11LOGO version is case-sensitive."

Reply:
I agree.